Url refactor.

WlodekLipski opened the pull request:
Index page, table and plot pages added. Small pages refactor done.
Need to connect value from dropdown list with data on table page and
plot.

WlodekLipski committed:
Url refactor.

Index page, table and plot pages added. Small pages refactor done.
Need to connect value from dropdown list with data on table page and
plot.

mikitachab commented on measurements/urls.py:
```diff
@@ -7,5 +7,6 @@
     path('', include(router.urls)),
-    path('table/', views.table)
+    path('view/', views.view),
+    path('data/', views.table)
```
why not just:
```python
path('table/', views.table)
```

mikitachab commented on measurements/views.py:
```diff
@@ -1,26 +1,50 @@
+        }
+    return render(request, "data_table.html", context)
+
+def view(request):
```
please consider more descriptive name

mikitachab commented on prepare_venv.sh:
```diff
@@ -1,5 +1,5 @@
 #!/bin/bash
-python3 -m virtualenv venv
-source venv/bin/activate
+python3 -m venv virtualenv
```
I don't understand why you did that. `venv` and `virtualenv` a different packages. they are doing mostly the same, `venv` is prebuilt, but `virtualenv` more preferable to use and `venv` is more preferable name for virtual environment. if you had troubles with running that script please try:
```bash
pip install virtualenv
```
and please change it as it was before.

WlodekLipski committed:
Update to pull request.

venv -> virtualenv
Name for plot and table updated.
Path to both pages now: ~/data/table or ~/data/plot

WlodekLipski commented:
Commit changes.
